### ISSUE TITLE
Passing Click through transparent Element (GUI)

### ISSUE THREAD

**Issue by Fanvaron** [xamarin/urho#244]
Hello,
I am working with the GUI and I have several layer, where each layer has a transparent fullscreen root.
The Problem now is, that only the top most layer gets click events through the normal handler (UI.UIMouseClick)
The other layers are sibling and thus they and their children get blocked from the topmost layers transparent root.

Is there a way to pass through click events, without manually searching for the next element underneath the click and handling it there. Which I'm trying to do but it gets more and more complicated, especially since I don't know how to trigger o.g. the Dropdown elements native click handler.

So basically 3 questions:
- Can I "pass on" a Click Event after receiving it
- Can I make an element "invisible" to click Events, so that it automatically goes through to elements underneath
- If those are not possible, can I fake a click event so that I can trigger the elements native Response

sincerely
Marvin
